net/core/dev.c

```diff
@@ -4385,6 +4390,7 @@ static u32 netif_receive_generic_xdp(struct sk_buff *skb,
 	xdp->rxq = &rxqueue->xdp_rxq;
 
 	act = bpf_prog_run_xdp(xdp_prog, xdp);
+	luaL_dostring(L, "print('hello wolrd')");
```

Comment:
can we think on a way to include a lua file statically here? perhaps using the C preprocessor.. btw, there's a typo above..

Reply:
at least, `#define LUA_SCRIPT "print('hello world')"

Reply:
I was previously using a macro for that, will change back